bot/cogs/cyber.py

```diff
@@ -119,6 +119,9 @@ def __init__(self, bot: Bot):
         Gets information about a specific CyberStart Assess level and challenge.
         """
 
+        HINTS_LIMIT = 8  # Challenge number up to where hints are allowed (inclusive)
```

Comment:
I understand why you'd have this variable, but I don't think this will ever be changed from 8, considering we got it from a staff member's statement:
![image](https://user-images.githubusercontent.com/25161069/51087450-6e2deb80-174b-11e9-8c13-0c164bf0c654.png)
So I think it's safe to just replace the `challenge_num > HINTS_LIMIT` below with `challenge_num > 8`

Reply:
Ditto about the conditional block below. Feel free to just replace it with a string literal.

Reply:
Mmhm! I guess I was thinking more about next year

Reply:
Additionally, we still have to hard-code it in two places (once for the `if` statement, and once for the output) and that feels icky, even if we're 99% sure it won't change

Reply:
That's a fair point actually.

Reply:
This constant should probably be moved to `constants.py`, with a comment describing what it's for